hydra_agent/graphutils.py

```diff
@@ -0,0 +1,95 @@
+from redisgraph import Node, Edge, Graph
+
+
+class GraphUtils:
+
+    def __init__(self, redis_proxy, graph_name='apidoc'):
+        self.redis_proxy = redis_proxy
+        self.redis_connection = redis_proxy.get_connection()
+        self.graph_name = graph_name
+        self.redis_graph = Graph("apidoc", redis_proxy)
+
+    def read(self, match, ret, where=None):
+        """
+        Run query to read nodes in Redis and return the result
+        :param match: Relationship between queried entities.
+        :param ret: Defines which property/ies will be returned.
+        :param where: Used to filter results, not mandatory.
+        :return: Corresponding Nodes
+        """
+        query = "MATCH(p:{})".format(match)
+        if where is not None:
+            query += " WHERE(p.{})".format(where)
+        query += " RETURN p.{}".format(ret)
+
+        return self.redis_connection.execute_command("GRAPH.QUERY",
+                                                     self.graph_name,
+                                                     query)
+
+    def update(self, match, set, where=None):
+        """
+        Run query to read nodes in Redis and return the result
```

Comment:
wrong docstring

Reply:
Thanks for all the reviews, will be picking them up.